### PR TITLE
Participant Course/Dates 

### PR DESCRIPTION
**Pull request by AsalaKM**
**Participant Course Details Page**

> Page that view the course details that related to the individual participant .

- Add /participants/course/details rout #44 
- Create page front end with css #45 

**Add Course for Participant Page**

> Page that allow to add new course for individual participant .

- Add /participants/course/add rout #56 
- Create page front end with css #57 

**Participant Date Details Page**

> Page that view the date details that related to the individual participant .

- Add /participants/date/details rout #48
- Create page front end with css #49 

**Add Date Page**

> Page that allow to add new date for individual participant .

- Add /participants/date/add rout #54 
- Create page front end with css #55

### CODE REVIEW

**Comment by ishak52** [comment]
@AsalaKM

File: src/client/src/components/pages/participants/addParticipantCourse/staticData.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            width: "20rem",
+            type: "date",
+        }
+    ],
+    [
+        {
+        tag: 'Button',
+        value: "Add Course",
+        color: "#272727",
+    },
```
Comment (ishak52) [inline src/client/src/components/pages/participants/addParticipantCourse/staticData.js]: You need to add `staticField: true` to make the buttons appear in your form

Comment (AsalaKM) [reply src/client/src/components/pages/participants/addParticipantCourse/staticData.js]: it was appeared in my form, 
there is no need of use it in my pages

Comment (AsalaKM) [reply src/client/src/components/pages/participants/addParticipantCourse/staticData.js]: I solve it now

Comment (ishak52) [reply src/client/src/components/pages/participants/addParticipantCourse/staticData.js]: :+1:

**Comment by AsalaKM** [comment]
Push in other branch #96

**Pull request closed without merging**